src/cmocka/rnp_tests_support.c

```diff
@@ -56,6 +56,14 @@ file_exists(const char *path)
     return stat(path, &st) == 0 && S_ISREG(st.st_mode);
 }
 
+/* Check if a directory exists */
+int
```

Comment:
We don't use bool anywhere, that's why I'm using int here, while it returns only 'true'/'false'